spec/models/subject_set_spec.rb

```diff
@@ -70,7 +70,7 @@
       end
     end
 
-    describe "#set_member_subjects_count" do
+    describe "#set_member_subjects_count", :disabled do
```

Comment:
does this still need to be disabled with the `after(:create)` block in the SMS factory?